app/src/main/java/com/fmsys/snapdrop/UpdateUtils.java

```diff
@@ -0,0 +1,112 @@
+package com.fmsys.snapdrop;
+
+import android.app.Activity;
+import android.app.DownloadManager;
+import android.app.ProgressDialog;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Environment;
+
+import androidx.core.content.FileProvider;
+
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.Request;
+import com.squareup.okhttp.Response;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.File;
```

Comment:
Codacy found an issue: [Wrong order for 'java.io.File' import.](https://app.codacy.com/gh/fm-sys/snapdrop-android/pullRequest?prid=6339390)